src/pages/Browse/Browse.tsx

```diff
@@ -19,13 +20,23 @@ export const Browse = () => {
     navigate('/add-model');
   }
 
+  function disableButton() {
+    return isOwnerOrAdmin();
+  }
+
   return (
     <>
       <Styled.BrowseWrapper>
         <Typography variant="h1">Browse all models</Typography>
-        <div className="btn-div">
-          <Button onClick={navigateAddModel}>Add new model</Button>
-        </div>
+        {disableButton() ? (
+          <div className="btn-div">
+            <Button disabled={!disableButton()} onClick={navigateAddModel}>
+              Add new model
+            </Button>
+          </div>
+        ) : (
+          <></>
+        )}
```

Comment:
```suggestion
        {isOwnerOrAdmin() ? (
          <div className="btn-div">
            <Button disabled={!isOwnerOrAdmin()} onClick={navigateAddModel}>
              Add new model
            </Button>
          </div>
        ) : (
          <></>
        )}
```
Is this function nessesary? Might be cleaner and easier to understand by using isOwnerOrAdmin() directly?